tests/test-dirs/stack-or-heap.t/records.ml

```diff
@@ -0,0 +1,35 @@
+(* Cursor is inside a record constructor (some of which don't allocate) *)
```

Comment:
Could you please add a test where a function takes its argument locally and the caller doesn't write the word `local_` on the argument?

```
let f (local_ _) = ()

let g x =
  f (Some x);
  f (local_ Some x);
  (* bonus: in tail position, `Some x` should be heap allocated *)
  f (Some x)
```